tools/test_deleted_renamed_referenced_files

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
+    if [ -f "$FILE" ] 
+    then
+        # if file exists it means it was renamed, and then original file name is retrieved by git log
+        FILE=$(git log --follow -p $FILE | grep 'rename from test' | awk '{print $3}')
+    fi
+    if [ -n "$(echo $FILE | grep '\.pm$')" ] 
+    then
+	# In case file is a module, module name appears in scheduling files excluding 'tests/' and file extension
+	file_to_verify=$(echo $FILE | sed -E 's/^tests\/(.*)\.pm$/\1/g')
```

Comment:
Let's fix indentation on these 2 lines too.

Reply:
For some weird reason, these two line appear to be a bit to the left compared to rest, but when I press "view file" they look aligned with the rest. In my local repo they also look aligned.

Reply:
Looks like it's tab vs spaces issue. On the lines above you have spaces and these 2 have tabs. Please, use one of two and use it consistently in the script. We also can have 4 hours discussion about tabs vs spaces and if tab should be 2 or 4 spaces =D

Reply:
Ahh.. no, there are much better ways to spend 4 hours. So Indeed, it looks like when using tab in particular page "Files changes" of the pull request, indentation looks different. So I used space everywhere.